fulp_modules/features/antagonists/bloodsuckers/code/rulesets/monsterhunter_event.dm

```diff
@@ -36,3 +36,64 @@
 		all_players.mind.add_antag_datum(/datum/antagonist/monsterhunter)
 		message_admins("MONSTERHUNTER NOTICE: [all_players] has awoken as a Monster Hunter.")
 		break
+
+//gives monsterhunters an icon in the antag selection panel
+/datum/dynamic_ruleset/midround/monsterhunter
+	name = "Monster Hunter"
+	antag_datum = /datum/antagonist/monsterhunter
+	antag_flag = ROLE_MONSTERHUNTER
+	weight = 0
+	cost = 200
+	protected_roles = list(
+		JOB_CAPTAIN,
+		JOB_DETECTIVE,
+		JOB_HEAD_OF_PERSONNEL,
+		JOB_HEAD_OF_SECURITY,
+		JOB_PRISONER,
+		JOB_SECURITY_OFFICER,
+		JOB_WARDEN,
+	)
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG,
+		ROLE_POSITRONIC_BRAIN,
+	)
+	required_candidates = 1
+	requirements = list(10,10,10,10,10,10,10,10,10,10)
+
+
+/datum/dynamic_ruleset/midround/monsterhunter/trim_candidates()
+	..()
+	for(var/mob/living/player in living_players)
+		if(issilicon(player))
+			living_players -= player
+		if(is_centcom_level(player.z))
+			living_players -= player
+		if((player.mind?.special_role || player.mind?.antag_datums?.len))
+			living_players -= player
+
+/datum/dynamic_ruleset/midround/monsterhunter/ready(forced = FALSE)
+	var/cancel = TRUE
+	for(var/datum/antagonist/monster in GLOB.antagonists)
```

Comment:
this is checking for all antagonists to make an antag a monster hunter, not humans

Reply:
shut the fuck up

Reply:
shut the fuck up